tests/ovirt/basic_vm_import_negative_test.go

```diff
@@ -0,0 +1,91 @@
+package ovirt_test
+
+import (
+	. "github.com/kubevirt/vm-import-operator/tests/matchers"
+	"github.com/kubevirt/vm-import-operator/tests/ovirt/vms"
+	"github.com/kubevirt/vm-import-operator/tests/utils"
+	sapi "github.com/machacekondra/fakeovirt/pkg/api/stubbing"
+	v1 "kubevirt.io/client-go/api/v1"
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+
+	fwk "github.com/kubevirt/vm-import-operator/tests/framework"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type basicVmImportNegativeTest struct {
+	framework *fwk.Framework
+}
+
+var _ = Describe("VM import", func() {
+
+	var (
+		f         = fwk.NewFrameworkOrDie("basic-vm-import-negative")
+		secret    corev1.Secret
+		namespace string
+		test      = basicVmImportNegativeTest{f}
+		err       error
+	)
+
+	BeforeEach(func() {
+		secret, err = f.CreateOvirtSecretFromBlueprint()
+		if err != nil {
+			Fail("Cannot create secret: " + err.Error())
+		}
+		namespace = f.Namespace.Name
+		err = f.OvirtStubbingClient.Reset("static-sso,static-transfer,static-namespace,static-transfers")
```

Comment:
It is already in the common BeforeEach (https://github.com/kubevirt/vm-import-operator/blob/ec9095c98ca3b4c627ad68511871611508ea9cb0/tests/framework/framework.go#L176)